pages/[id].js

```diff
@@ -78,8 +79,18 @@ export default function ProfilePage({ entries = [] }) {
           {age - 1 === 1 ? ` Jahr alt` : ` Jahre alt`}
         </p>
       </StyledBirthday>
+      <MoreInfoForm
+        personId={id}
+        currentProfile={currentProfile}
+        id={currentProfile.id}
+        name={currentProfile.name}
+        birthday={currentProfile.birthday}
+        ideas={currentProfile.ideas}
+        notes={currentProfile.notes}
```

Comment:
du kannst alternativ auch nur currentProfile mitgeben, damit du nicht so viele Props hast. da sind ja alle Infos drin. auf der Gegenseite kannst du dann destructuren `const {id, name, birthday, ideas, notes} = currentProfile` . und brauchst du zwei mal die ID?

edit: du nimmst in der MoreInfoForm.js die props auch nicht entgegen, also kannst du die löschen

Reply:
Danke, stimmt! Hab id, name, birthday und ideas rausgenommen 👍

Reply:
und notes jetzt auch, nachdem ich deinen Vorschlag weiter unten angenommen habe :)